libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -49,14 +50,20 @@ export class BookSearchComponent implements OnInit {
 
   searchExample() {
     this.searchForm.controls.term.setValue('javascript');
-    this.searchBooks();
+    this.searchForBooks();
   }
 
-  searchBooks() {
+  searchForBooks() {
     if (this.searchForm.value.term) {
       this.store.dispatch(searchBooks({ term: this.searchTerm }));
     } else {
       this.store.dispatch(clearSearch());
     }
   }
+
+  instantSearch(searchString) {
```

Comment:
This is clunky. There's a better, more reactive way to do this. Perhaps there's a stream of changes we can use...?